src/components/common/bullets-colin-link.vue

```diff
@@ -0,0 +1,175 @@
+import { Getter } from 'vuex-class'
+import { CompanyType, EntityType } from '@/enums'
+import NameInput from '@/components/new-request/name-input.vue'
+import { Navigate } from '@/plugins'
+import { NrAffiliationMixin } from '@/mixins'
+import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module'
+
+@Component({
+  components: {
+    NameInput
+  }
+})
+export default class BulletsColinLink extends Mixins(NrAffiliationMixin) {
+  /** The selected business type. */
+  @Prop({ default: '' }) readonly businessType!: EntityType
+
+  /** Show Colin Button. */
+  @Prop({ default: false }) readonly colinButton!: boolean
+
+  /** Show Designation Dropdown. */
+  @Prop({ default: false }) readonly showDesignation!: boolean
+
+  /** Whether user is authenticated. */
+  @Getter getIsAuthenticated!: boolean
+
+  // Local properties
+  selectedCompanyType: CompanyType = null
+  bulletPoints = [
+    'Your business name will be the Incorporation Number assinged by the Registry.',
+    'You can change your business name at a later date.',
+    'It is not possible to request a specific Incorporation Number.'
+  ]
+  readonly colinLink = sessionStorage.getItem('CORPORATE_ONLINE_URL')
+
+  // For template
+  readonly CompanyType = CompanyType
+
+  /**
+   * The alternate codes for entity types.
+   * Alternate codes are used in Entities UIs.
+   */
+  entityTypeAlternateCode (entityType: EntityType): CorpTypeCd {
+    switch (entityType) {
+      case EntityType.BC: return CorpTypeCd.BENEFIT_COMPANY
+      case EntityType.CC: return CorpTypeCd.BC_CCC
+      case EntityType.CR: return CorpTypeCd.BC_COMPANY
+      case EntityType.UL: return CorpTypeCd.BC_ULC_COMPANY
+      default: return null
+    }
+  }
+
+  /** Navigate to the Entity Dashboard. */
+  goToEntityDashboard (businessId: string): void {
+    if (businessId) {
+      const dashboardUrl = sessionStorage.getItem('DASHBOARD_URL')
+      Navigate(`${dashboardUrl}${businessId}`)
+    }
+  }
+
+  /**
+   * If user is authenticated, create draft business and redirect to Dashboard.
+   * If user is not authenticated, redirect to login screen then redirect back.
+   */
+  async incorporateNowClicked () {
+    const legalType = this.entityTypeAlternateCode(this.businessType)
+    if (this.getIsAuthenticated) {
+      await this.incorporateNow(legalType)
+    } else {
+      // persist legal type of incorporate now in session upon authentication via Signin component
+      sessionStorage.setItem('LEGAL_TYPE', legalType)
+      // navigate to BC Registry login page with return parameter
+      const registryHomeUrl = sessionStorage.getItem('REGISTRY_HOME_URL')
+      const nameRequestUrl = `${window.location.origin}`
+      Navigate(`${registryHomeUrl}login?return=${nameRequestUrl}`)
+    }
+  }
+
+  /** Emit the selected radio button CompanyType enum value. */
+  @Emit('radioButtonChange')
+  radioButtonChanged (event: CompanyType): CompanyType {
+    return event
```

Comment:
FYI, there's an alternate syntax that does the same thing:
```
@Emit('radioButtonChange')
radioButtonChanged (event: CompanyType): void {}
```

Reply:
I like this 😄 I changed it!